### PR TITLE
[16] Render the related product on HTML page with JavaScript

### PR DESCRIPTION
**Pull request by Cynthia-Berry**
## Task: `[16] Render the related product on HTML page with JavaScript`
- `Did task on working with array`

### CODE REVIEW

**Review by adaugochi — COMMENTED** [review]
Well done

You can only run one of your loop. For your own reference case. just comment out the rest so you can still be about to make reference to them

File: submissions/cynthiaNwakaeme/JavaScript/script.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+// Using the While Loop
+var product = 0;
+while (product < productList.length){
+    console.log(productList[product].productName + " with " + productList[product].imageUrl + " is sold for " + productList[product].productPrice + ".");
+    product++;
+};
+
+// Using the Do ... While Loop
+var product = 0;
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/JavaScript/script.js:33]: You don't need to declare `product = 0` again since you have done that in line 26

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Great Job @Cynthia-Berry

**Review by adaugochi — APPROVED** [review]

File: submissions/cynthiaNwakaeme/JavaScript/script.js (hunk 1 above)
Comment (Cynthia-Berry) [reply submissions/cynthiaNwakaeme/JavaScript/script.js]: Thank You ma. I commented out the code for other loops

**Pull request merged**